test_average (__main__.TestExercice) ... ok
test_square (__main__.TestExercice) ... ok
test_square_root (__main__.TestExercice) ... ok
test_to_celsius (__main__.TestExercice) ... ok
test_to_degrees (__main__.TestExercice) ... ERROR
test_to_farenheit (__main__.TestExercice) ... ok
test_to_radians (__main__.TestExercice) ... ok

======================================================================
ERROR: test_to_degrees (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 80, in test_to_degrees
    output = [exercice.to_degrees(v) for v in values]
  File "test_exercice.py", line 80, in <listcomp>
    output = [exercice.to_degrees(v) for v in values]
  File "/home/runner/work/c04-ch3-exercices-homer-crypto/c04-ch3-exercices-homer-crypto/exercice.py", line 33, in to_degrees
    return (degress - min), (min - sec), sec
NameError: name 'degress' is not defined

----------------------------------------------------------------------
Ran 7 tests in 0.001s

FAILED (errors=1)
